src/Constants/funding-option.js

```diff
@@ -0,0 +1,7 @@
+const RECOMMEND_OPTION = [
+  ['Latest', 0],
+  ['Time Limit', 1],
+  ['Popular', 2],
+];
```

Comment:
요게 select로 Recommend(추천순) / Latest(최신순) / Time Limit(남은 시간 순) / Popular(인기순) 옵션인 것으로 알아요 !! ㅎㅎ
```suggestion
const FILTER = [
  ['Recommended', 0],
  ['Latest', 1],
  ['Time Limit', 2],
  ['Popular', 3],
];
```

Reply:
아 선택된게 그 메뉴바로 보이는거군요 
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ ㅋㅋㅋ별 생각 없이 보이는대로 짜다보니.......

Reply:
헿 고치믄 말해주셔용 ~~!~!

Reply:
> 헿 고치믄 말해주셔용 ~~!~!

suggested change 다 수정했습니다~!